webshare/share-url-noscheme-manual.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>WebShare Manual Test: Share with URL without a scheme</title>
```

Comment:
Hmm, so I suppose it actually does not matter because they can only be run manually. When automated, however, they'll need to be over HTTPS, so -manual.https.html SGTM since they already exist. (Hypothetically, one could make a harness for manual runs where the user just sits and clicks.)